app/routes/questionnaire.py

```diff
@@ -361,6 +371,30 @@ def get_thank_you(schema, session_store, questionnaire_store):
     )
 
 
+@post_submission_blueprint.route("view-response/", methods=["GET"])
+@with_session_store
+@with_questionnaire_store
+@with_schema
+def get_submitted_response(schema, questionnaire_store, session_store):
+    try:
+        submitted_response = SubmittedResponse(
+            schema,
+            questionnaire_store,
+            session_store.session_data,
+            flask_babel.get_locale().language,
+        )
+
+    except (SubmittedResponseNotEnabled, SubmittedResponseExpired):
```

Comment:
The expected behaviour has still not be fulled designed, it was agreed this was fine until the new page was created. This also applies to NoQuestionnaireState (with/without cookie)